crypto/keys/src/traits.rs

```diff
@@ -195,7 +231,7 @@ where
 pub trait Kex {
     type Public: KexPublic<KexEphemeralPrivate = Self::EphemeralPrivate>
         + for<'reusable> From<&'reusable Self::Private>
-        + AsRef<[u8]>;
+        + GetBytes;
```

Comment:
The motivation for this change was so that `RistrettoPublic` can satisfy `Kex::Public`.